app/routes/wow.ilvl-shopping-list.tsx

```
@@ -0,0 +1,368 @@
+}
+
+type LoaderResponseType =
+  | {}
+  | { exception: string }
+  | (IlvlWoWListResponse & {
+      sortby: string
+      formValues: { itemID: number; maxPurchasePrice: number; desiredMinIlvl: number; desiredStats: ItemStat[] }
+    })
+
+type ActionResponseType =
+  | {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)